FindWordPosition.java

```diff
@@ -0,0 +1,24 @@
+package hw5;
+
+public class FindWordPosition {
+
+
+    public static int findWordPosition(String source, String target) {
```

Comment:
The same here, please add null validation for both 'source' and 'target' strings